### DISCUSSION TITLE
How to change the image of init/wait container pull mode

### DISCUSSION THREAD

**Discussion by xiongyunhua-star** [argoproj/argo-workflows#7342] [Q&A] [answered]
How to implement workflow loading in private harbor warehouse Init and  wait container and set pull policy?

Yaml  defaults:
    image: quay.io/argoproj/argoexec:latest
    imagePullPolicy: Always
    name: init

**Comment by xiongyunhua-star** [comment]
update  controller-pod

**Comment by terrytangyuan** [comment, marked as answer]
You can use PodSpecPatch in your workflow or specify your executor container spec in controller configmap.

**Reply by xiongyunhua-star** [reply]
? for example?  how to set argoexec:latest-images  pull policy ,   I want to get IfNotPresent

**Reply by xiongyunhua-star** [reply]
I  has resolved,  by adding the args  --executor-image-pull-policy string   Executor imagePullPolicy to use (overrides value in configmap)
**
